setup.py

```diff
@@ -13,7 +13,7 @@
       package_dir={'': 'src'},
       packages=find_packages('src'),
       py_modules=['protego'],
-      python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*',
+      python_requires='>=3.6',
```

Comment:
We could as well drop support for 3.6 at this point, as we ([you!](https://github.com/scrapy/scrapy/pull/5514) :smile:) did in Scrapy.